step-definitions/core/core.js

```diff
@@ -3,7 +3,7 @@ import { Given, When, Then } from 'cucumber';
 
 import { setupEvan } from '../../test-utils/test-utils.js';
 
-export const WAIT_TIME = 15 * 1000;
+export const WAIT_TIME = 300 * 1000;
```

Comment:
that's a loooooooooooooooooooooong time. Meaning a test will fail just after 300 Secs. Also the slowest browser and network connection shouldn't take that long. :/

Reply:
but eve buy duration can be that long ;-)

Reply:
maybe should add a "wait till loading state completed" function with an optional duration param?

Reply:
something like this: `Then the spinner should disappear after N seconds`